doc/source/whatsnew/v0.25.0.rst

```diff
@@ -567,6 +567,7 @@ Other API changes
 - Using an unsupported version of Beautiful Soup 4 will now raise an ``ImportError`` instead of a ``ValueError`` (:issue:`27063`)
 - :meth:`Series.to_excel` and :meth:`DataFrame.to_excel` will now raise a ``ValueError`` when saving timezone aware data. (:issue:`27008`, :issue:`7056`)
 - :meth:`DataFrame.to_hdf` and :meth:`Series.to_hdf` will now raise a ``NotImplementedError`` when saving a :class:`MultiIndex` with extention data types for a ``fixed`` format. (:issue:`7775`)
+- :meth:`Categorical.ravel` will now return a :class:`Categorical` instead of a NumPy array. (:issue:`27153`)
```

Comment:
If we want this for 0.25 we should make it in its own PR.

Is it hard to deprecate? Do you have downstream changes that rely on EA.ravel() being an EA? (probably)

Reply:
> Do you have downstream changes that rely on EA.ravel() being an EA? (probably)

Yes, upcoming changes in core.internals definitely rely on this.

> If we want this for 0.25 we should make it in its own PR.
> Is it hard to deprecate? 

I like this idea; anything we can do to make a destined-to-be-big PR smaller is worthwhile.

Reply:
I am not averse to *some* of these changes in 0.25, especially any breaking changes that we really need (but these have yet to be fully accepted), but this is not blocking for 0.25.

Reply:
FWIW, we have zero tests for current Categorical.ravel behavior.  OTOH, I'm just about ready with a branch that deprecates the behavior (and implements no-op ravel for EA)